src/basic/string-util.c

```diff
@@ -791,10 +791,10 @@ char *strip_tab_ansi(char **ibuf, size_t *_isz, size_t highlight[2]) {
         return *ibuf;
 }
 
-char *strextend_with_separator(char **x, const char *separator, ...) {
-        bool need_separator;
+char *strextend_with_separator_internal(char **x, const char *separator, ...) {
         size_t f, l, l_separator;
-        char *r, *p;
+        bool need_separator;
```

Comment:
switching to reverse christmas tree notation? 'tis the season, I guess :-D

Reply:
well, i usually do reverse christmas tree, without thinking much. we don't require it, but i still like it enough to do it...

Reply:
it's perfectly fine, I just never noticed :-)